pkg/providers/instance/azure_client.go

```diff
@@ -193,7 +194,7 @@ func NewAZClient(ctx context.Context, cfg *auth.Config, env *azclient.Environmen
 
 	// TODO: this one is not enabled for rate limiting / throttling ...
 	// TODO Move this over to track 2 when skewer is migrated
```

Comment:
Remove this `TODO` comment?
```suggestion
```

Reply:
As above, the actual SDK is still track1, so I think this comment still applies.